File: shivani/src/main/java/org/example/Employee.java

import java.time.LocalDate;
public class Employee {
    int id;
    String name;
    Address address;
    int age;
    LocalDate joiningDate;
    double salary;

    public Employee(int id, String name, Address address, int age, LocalDate joiningDate, double salary) {
        this.id = id;
        this.name = name;
        this.address = address;
        this.age = age;
        this.joiningDate = joiningDate;
        this.salary = salary;
    }

    public double getSalary() {
        return salary;
    }

    @Override
    public String toString() {
        return "Employee{id=" + id + ", name='" + name + "', address=" + address + ", age=" + age + ", joiningDate=" + joiningDate + ", salary=" + salary + "}";
    }
}

File: Vishal/Assignment 1/Employee.cs
using System;
namespace EmployeeManagement
{
    public class Employee
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public Address Address { get; set; }

        public int Age { get; set; }
        public DateTime JoiningDate { get; set; }
        public decimal Salary { get; set; }

        public Employee(int id,string name,Address address,int age,DateTime joiningDate,decimal salary)
        {
            Id = id;
            Name = name;
            Address = address;
            Age = age;
            JoiningDate = joiningDate;
            Salary = salary;
        }

        public override string ToString()
        {
            return $"Id: {Id}, Name: {Name}, Address: {Address}, Age: {Age}, Joining Date: {JoiningDate.ToShortDateString()}, Salary: {Salary:C}";
        }

    }
}
